job/validators.py

```diff
@@ -209,42 +209,43 @@ def cross_clean(self):
         PV tilt set to latitude if not provided and prod_factor_series validated
         """
         def cross_clean_pv(pvmodel):
-            if pvmodel.__getattribute__("tilt") == 0.537:  # 0.537 is a dummy number, default tilt
-                pvmodel.__setattr__("tilt", self.models["Site"].__getattribute__("latitude"))
+            if pvmodel.__getattribute__("tilt") == 0.537:  # 0.537 is a dummy number, default tilt # TODO: change to None? 
+                pvmodel.__setattr__("tilt", self.models["Site"].__getattribute__("latitude")) # TODO differentiate default tilt based on array_type
+            # TODO add cross_clean for azimuth
```

Comment:
@rathod-b  We should also follow the Julia Pkg default behavior for PV azimuth:
`azimuth::Real = latitude≥0 ? 180 : 0, # set azimuth to zero for southern hemisphere` 

Could you make this update as well?

Reply:
updated